pangraph.cabal

```diff
@@ -18,15 +18,15 @@ tested-with:
 library
   hs-source-dirs:      src
   exposed-modules:     Pangraph
-  other-modules:       Parsers.GmlP
-                     , Parsers.Discrim
+  other-modules:       Parsers.GmlParser
```

Comment:
On a second thought, it probably make sense to use this naming convention: `Parser.GraphML` -- no need to repeat the word `Parser` twice. Also note, that usually Haskell modules are not plural, so `Parser`, not `Parsers`.